src/bun.js/bindings/bindings.cpp

```diff
@@ -2462,52 +2441,37 @@ bool JSC__JSValue__isSameValue(JSC__JSValue JSValue0, JSC__JSValue JSValue1,
     return JSC::sameValue(globalObject, left, right);
 }
 
+#define IMPL_DEEP_EQUALS_WRAPPER(strict, enableAsymmetricMatchers, globalObject, a, b) \
```

Comment:
For future reference, you can use a `static inline bool` here and it'll compile to almost the same thing

Reply:
fixed